velox/exec/Exchange.h

```diff
@@ -352,15 +352,18 @@ class Exchange : public SourceOperator {
       int32_t operatorId,
       DriverCtx* FOLLY_NONNULL ctx,
       const std::shared_ptr<const core::ExchangeNode>& exchangeNode,
-      std::shared_ptr<ExchangeClient> exchangeClient)
+      std::shared_ptr<ExchangeClient> exchangeClient,
+      const std::string& operatorType = "Exchange",
+      VectorSerde* FOLLY_NONNULL serde = getVectorSerde().get())
```

Comment:
perhaps, remove default values and update the callers to specify these values explicitly